tasks/libs/pipeline_notifications.py

```diff
@@ -125,9 +126,9 @@ def base_message(header, state):
 {enhanced_commit_title} (<{commit_url_gitlab}|{commit_short_sha}>)(:github: <{commit_url_github}|link>) by {author}"""
 
 
-def get_git_author():
+def get_git_author(format='an'):
```

Comment:
Not sure this is really useful to keep the format as a parameter. If you use another string for `format` the function can be no longer returning the author of the PR (`H` for example)